auth-web/src/components/auth/staff/continuation-application/ContinuationApplicationTable.vue

```diff
@@ -97,6 +97,42 @@
                         hide-details="auto"
                       />
 
+                      <!-- Date Picker to select date submitted range -->
+                      <div v-else-if="['submissionDate'].includes(header.value)">
+                        <v-text-field
+                          :value="reviewParams.startDate ? 'Custom':''"
+                          filled
+                          :placeholder="'Date Submitted'"
+                          readonly
+                          dense
+                          hide-details="auto"
+                          @click="showDatePicker = true"
+                        >
+                          <template #append>
+                            <v-icon
+                              v-if="reviewParams.startDate"
+                              color="primary"
+                              @click="updateDateRange({ startDate: '', endDate: '' })"
+                            >
+                              mdi-close
+                            </v-icon>
+                            <v-icon color="primary">
+                              mdi-calendar
+                            </v-icon>
+                          </template>
+                        </v-text-field>
+
+                        <DatePicker
+                          v-show="showDatePicker"
+                          ref="datePicker"
+                          :reset="dateRangeReset"
+                          :style="{ height: tableDataOptions.itemsPerPage === 5 ? '200px' : 'auto' }"
```

Comment:
When items per page =5, the continuation table height is reduced, so I adjust the height of the date picker and it is scrollable now

Reply:
I don't think it would be a problem to  keep this date picker "auto" height even when the table shows only 5 items. Double-check with Janis.

Reply:
If we keep this date picker "auto" height when the table shows only 5 items:
![image](https://github.com/user-attachments/assets/954fc1e8-4c75-4dad-bc01-1b63361ec991)
Half of the date picker is hidden. The user can only change the items per page option to 10 to see it.

Reply:
Looks like it's "under" the table. You may be able to fix this using `z-index`, or perhaps attaching it to the section div instead of the table...?

Reply:
Updated

Reply:
Looks much better. Please resolve this conversation if there's nothing else here to discuss.